<p align="center">
  <img src="image\git_img_top.png" width="250" height="100" />
</p>
<h2 align="center">Community curated database of oil metagenomes</h2>

<div align="center">
  
  ![check_dataset paasing](https://img.shields.io/badge/check__dataset-passing-brightgreen)
  ![Latest Release](https://img.shields.io/badge/Latest__Release-v0.1-orange)
  
</div>
 
:octocat: Community curated database of the metagenome of oil and oil fieldsüõ¢Ô∏èü¶†

+ [Description](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#description)
+ [Usage](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#usage)
+ [Samples Column Specifications](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#samples-column-specifications)
+ [Libraries Column Specifications](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#libraries-column-specifications)
+ [Contributing](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#contributing)
  + [Contributing your own Publications](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#contributing-your-own-publications)
  + [Contributing previously published publications](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#contributing-previously-published-publications)
  + [Guidelines and Conventions](https://github.com/agni-bioinformatics-lab/OilMetagenomesDB/blob/main/README.md#guidelines-and-conventions)
  
## Description
–≠—Ç–æ—Ç —Å–ø–∏—Å–æ–∫ —Å–æ–¥–µ—Ä–∂–∏—Ç –∏–Ω—Ñ–æ—Ä–º–∞—Ü–∏—é –æ –º–µ—Ç–∞–≥–µ–Ω–æ–º–µ —Å—ã—Ä–æ–π –Ω–µ—Ñ—Ç–∏ –∏ –º–µ—Ç–∞–≥–µ–Ω–æ–º–∞ –æ–∫—Ä—É–∂–∞—é—â–µ–π —Å—Ä–µ–¥—ã –≤ –º–µ—Å—Ç–æ—Ä–æ–∂–¥–µ–Ω–∏—è—Ö –Ω–µ—Ñ—Ç–∏. –ó–¥–µ—Å—å –º–æ–∂–Ω–æ –Ω–∞–π—Ç–∏ –Ω–∞–∑–≤–∞–Ω–∏–µ –ø—É–±–ª–∏–∫–∞—Ü–∏–π, –≥–æ–¥—ã –ø—É–±–ª–∏–∫–∞—Ü–∏–π, DOI, —Ç–∏–ø –æ–±—Ä–∞–∑—Ü–æ–≤, –≥–µ–æ–≥—Ä–∞—Ñ–∏—á–µ—Å–∫–æ–µ –º–µ—Å—Ç–æ–ø–æ–ª–æ–∂–µ–Ω–∏–µ –≤–∑—è—Ç–∏—è –æ–±—Ä–∞–∑—Ü–∞, —Å–æ–ø—É—Ç—Å—Ç–≤—É—é—â–∏–µ —É—Å–ª–æ–≤–∏—è. 

## Usage
–ß—Ç–æ–±—ã –∑–∞–≥—Ä—É–∑–∏—Ç—å –ø–æ—Å–ª–µ–¥–Ω—é—é —Å—Ç–∞–±–∏–ª—å–Ω—É—é –≤–µ—Ä—Å–∏—é —Å–ø–∏—Å–∫–∞ –ø—Ä–∏–º–µ—Ä–æ–≤ OilMetagenomeBD –∏ –¥–æ–±–∞–≤–∏—Ç—å –¥–∞–Ω–Ω—ã–µ, –Ω–µ–æ–±—Ö–æ–¥–∏–º–æ –≤—ã–ø–æ–ª–Ω–∏—Ç—å —Å–ª–µ–¥—É—é—â–∏–µ —à–∞–≥–∏:
1. –ü–µ—Ä–µ–π—Ç–∏ –≤ —Ä–µ–ø–æ–∑–∏—Ç–æ—Ä–∏–π OilMetagenomeBD;
2. –°–∫–æ–ø–∏—Ä–æ–≤–∞—Ç—å SSH –∫–ª—é—á –≤–æ –≤–∫–ª–∞–¥–∫–µ "<> Code";
3. –û—Å–Ω–æ–≤–Ω—ã–µ –∫–æ–º–∞–Ω–¥—ã –¥–ª—è –∫–æ–º–∞–Ω–¥–Ω–æ–π —Å—Ç—Ä–æ–∫–∏:
* git - check if Git installed
* git clone <link> - cloning repo on your computer
* git status - check the changes
* git commit -m "add script.py with greeting" - save change in repository (-m is some message of what you‚Äôve done)
* git status - check that commit is created 
* git push - update the GitHub repository
## Samples Column Specifications
123
## Libraries Column Specifications
The libraries tables store information about each specific reed from the library - id_ in databases, sequencing type (paired-end, single-end), sequencing strategy (WGS, RNA-Seq, amplicon), links to downloads and publications, etc.

- üèû: oilfield environmental metagenomes
- ü¶†: crude oil metagenomes

Numeric fields (e.g. `read_count`), can be filled with `NA` to indicate 'no
reported value'. Free text fields (e.g. `library_name`) can be indicated with
`Unknown`, and restricted category columns sometimes will have an `unknown`
option.

All column with 'defined categories' should be validated against
`assets/enums/<column>.json`. This is to ensure data consistency. E.g., all
libraries sequenced on Illumina NextSeq 500s are listed as `NextSeq 500` (as
defined in `assets/enums/instrument_models.json`). This is to ensure data
consistency.

If you wish to a new category, please make a separate pull-request with your
modification in the corresponding `assets/enums/<column>.json` file.

Library columns are as follows:
* project name
* publication year
* data publication doi
* sample name
* archive
* archive project
* archive sample accession
* library name
* strand type
* library polymerase
* library treatment
* library concentration
* instrument model
* library layout
* library strategy
* amplicon variable region
* read count
* archive data accession
* download links
* download md5s
* download sizes

## Contributing
123
### Contributing your own Publications
123
### Contributing previously published publications
123
### Guidelines and Conventions
123
